script.py

```diff
@@ -49,8 +49,7 @@
 # If we don't have the date, update
 if update:
     # Prepare the dictionaire
-    dict_prices = {PREVIOUS_WEEK: curret_data[CURRENT_WEEK]}
-    dict_prices[CURRENT_WEEK] = {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: {}}
+    dict_prices = {PREVIOUS_WEEK: curret_data[CURRENT_WEEK], CURRENT_WEEK: {START_DATE_KEY: start_date, END_DATE_KEY: end_date, GAS_KEY: {}}}
```

Comment:
There is typo `curret_data`, should be `current_data`; this stems from here: https://github.com/carlosrsabreu/devo-abastecer/blob/010431a06eb908acff66d6f896f3b8f15e87dff9/script.py#L13-L15